packages/bits/src/lib/wizard-v2/wizard.component.less

```diff
@@ -145,7 +145,7 @@
 
 .nui-wizard-horizontal-content-container {
     overflow: hidden;
-    padding: 0 @wizard-side-gap @wizard-side-gap @wizard-side-gap;
+    padding: @wizard-side-gap;
```

Comment:
In the grooming we had on Thursday, we talked about not having padding in the content area.

Will this be addressed in a subsequent ticket?